src/cli_common.h

```diff
@@ -47,4 +48,16 @@ ssize_t cliWriteConn(redisContext *c, const char *buf, size_t buf_len);
  */
 int cliSecureInit();
 
+/* Create an sds from stdin */
+sds readArgFromStdin(void);
+
+/* Create an sds array from argv, either as-is or by dequoting every
+ * element. When quoted is non-zero, may return a NULL to indicate an
+ * invalid quoted string.
+ */
```

Comment:
traditionally in redis, the documentation is in the C file next to the implementation and the header file has just raw forward declarations.
I see there are already a few cloned comments in this file, i think we should trim all of them (to avoid duplicate and out of sync comments)